utils/_context/_scenarios/otel_collector.py

```diff
@@ -60,6 +64,61 @@ def configure(self, config: pytest.Config) -> None:
             "otel_collector", self.collector_container.image.labels["org.opencontainers.image.version"]
         )
 
+    def customize_feature_parity_dashboard(self, result: dict) -> None:
+        result["configuration"]["collector_version"] = str(self.library.version)
+
+        if hasattr(self.collector_container, "image") and self.collector_container.image:
+            result["configuration"]["collector_image"] = self.collector_container.image.name
+
+            # Extract image commit/revision if available from labels
+            if self.collector_container.image.labels:
+                image_labels = self.collector_container.image.labels
+                if "org.opencontainers.image.revision" in image_labels:
+                    result["configuration"]["collector_image_commit"] = image_labels[
+                        "org.opencontainers.image.revision"
+                    ]
+
+        # Parse OTel collector configuration file
+        config_file_path = Path(self.collector_config_file)
+        result["configuration"]["config_file"] = config_file_path.name
+
+        try:
+            with open(config_file_path, "r", encoding="utf-8") as f:
+                otel_config = yaml.safe_load(f)
+
+            if "receivers" in otel_config:
+                result["configuration"]["receivers"] = list(otel_config["receivers"].keys())
+                if "postgresql" in otel_config["receivers"]:
+                    pg_config = otel_config["receivers"]["postgresql"]
+                    result["configuration"]["postgresql_receiver"] = {
+                        "endpoint": pg_config.get("endpoint"),
+                        "databases": pg_config.get("databases", []),
+                    }
+
+            if "exporters" in otel_config:
+                result["configuration"]["exporters"] = list(otel_config["exporters"].keys())
+                if "datadog" in otel_config["exporters"]:
+                    dd_exporter_config = otel_config["exporters"]["datadog"]
+                    result["configuration"]["datadog_exporter_config"] = {
+                        "metrics": dd_exporter_config.get("metrics", {}),
+                    }
+
+            if "service" in otel_config and "pipelines" in otel_config["service"]:
+                result["configuration"]["pipelines"] = list(otel_config["service"]["pipelines"].keys())
+
+        except Exception as e:
+            logger.warning(f"Failed to parse OTel collector config: {e}")
```

Comment:
It should be an error : 

```suggestion
            pytest.exit(f"Failed to parse OTel collector config: {e}", 1)
```